### DISCUSSION TITLE
Why the Python API page is down?

### DISCUSSION THREAD

**Discussion by bseditor** [efroemling/ballistica#389] [Q&A] [answered]
Is this page moved to somewhere else?

**Comment by efroemling** [comment, marked as answer]
My bad; @Dliwk wired up some fancy new docs generation stuff and I still need to set up a cron job to autogenerate docs daily based on the latest code. I'm about to push the first 1.7 test builds (hopefully this week) and then I'll try to get docs stuff fixed right after.
In the meantime you should be able to look at older commits in this repo to see the old docs .md file.

**Comment by Delitel-WEB** [comment]
Also faced with this problem here are the docks from past comits https://github.com/efroemling/ballistica/blob/4d31d59467a4cae8140da47812a6e8fa7ae208ba/docs/ba_module.md
